### PR TITLE
pxt: theming for cloud projects card

### PR DESCRIPTION
**Pull request by eanders-ms**
Make the cloud projects card gray by default. Will be overridden by targets.

### CODE REVIEW

**Comment by kimprice** [comment]
If it will be overwritten, why even change to gray?

**Comment by eanders-ms** [comment]
> If it will be overwritten, why even change to gray?

It needs to be set to some default value. I chose a neutral one. It can help to indicate when a target hasn't yet themed this element.

File: webapp/public/vs/editor/editor.main.css [hunk 1]
```diff
@@ -3,4 +3,4 @@
...
...ng}.monaco-mouse-cursor-text{cursor:text}.hc-black.mac .monaco-mouse-cursor-text,.hc-black .mac .monaco-mouse-cursor-text,.vs-dark.mac .monaco-mouse-cursor-text,.vs-dark .mac .monaco-mouse-cursor-text{cursor:-webkit-image-set(url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAQAAAC1+jfqAAAAL0lEQVQoz2NgCD3x//9/BhBYBWdhgFVAiVW4JBFKGIa4AqD0//9D3pt4I4tAdAMAHTQ/j5Zom30AAAAASUVORK5CYII=) 1x,url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAQAAADZc7J/AAAAz0lEQVRIx2NgYGBY/R8I/vx5eelX3n82IJ9FxGf6tksvf/8FiTMQAcAGQMDvSwu09abffY8QYSAScNk45G198eX//yev73/4///701eh//kZSARckrNBRvz//+8+6ZohwCzjGNjdgQxkAg7B9WADeBjIBqtJCbhRA0YNoIkBSNmaPEMoNmA0FkYNoFKhapJ6FGyAH3nauaSmPfwI0v/3OukVi0CIZ+F25KrtYcx/CTIy0e+rC7R1Z4KMICVTQQ14feVXIbR695u14+Ir4gwAAD49E54wc1kWAAAAAElFTkSuQmCC) 2x) 5 8,text}.monaco-progress-container{width:100%;height:5px;overflow:hidden}.monaco-progress-container .progress-bit{width:2%;height:5px;position:absolute;left:0;display:none}.monaco-progress-container.active .progress-bit{display:inherit}.monaco-progress-container.discrete .progress-bit{left:0;transition:width .1s linear}.monaco-progress-container.discrete.done .progress-bit{width:100%}.monaco-progress-container.infi
```
Comment (kimprice) [inline webapp/public/vs/editor/editor.main.css:6]: Is this generated automatically?

Comment (eanders-ms) [reply webapp/public/vs/editor/editor.main.css:6]: Yep. I don't know when or why, but sometimes it gets regenerated. Whenever an auto-generated file changes locally, my policy is to just commit it.

**Review by kimprice — APPROVED** [review]

**Pull request merged**